cs/message.go

```diff
@@ -171,6 +171,8 @@ const (
 	PlayerMessagePlayerTechLevelGainedInvasion
 	PlayerMessagePlayerTechLevelGainedScrapFleet
 	PlayerMessagePlayerTechLevelGainedBattle
+	PlayerMessagePlayerMysteryTechGainedScrapFleet
+	PlayerMessagePlayerMysteryTechGainedBattle
```

Comment:
unfortunately, these have to be added at the end because they are incremented numbers. If you add them in the middle, all existing games will have messageIds that are incorrect.

At some point I'll convert these message types to strings so I can sort them, but that would involve a relatively complicated database upgrade step (because they're ints in the database as well...)

Reply:
Moved em to the end.